script/packtpub.py

```diff
@@ -124,6 +124,10 @@ def download_ebooks(self, types):
             for type in types]
 
         directory = self.__config.get('path', 'path.ebooks')
+
+        if self.__config.get('path', 'path.separate') == "true":
+            directory = directory + '/' + self.info['title'] + ' - ' + self.info['author']
```

Comment:
Moreover to concatenate the string you could use `.format()` or any equivalent to make it more readable